apps/cms/README.md

```diff
@@ -115,13 +115,13 @@ Note that once a Pack Template record is published, many of its fields cannot be
 
 ## Sets
 
-Sets are an optional mechanism to group NFTs in a way that can incentivize an end-user to collect all of the NFTs assigned to a Set. For example, let's imagine we are distributing football card NFTs. A Set might be all of the NFTs for a player on a given team in the 2022 season.
+Sets are an optional mechanism to group NFTs in a way that can incentivize an end-user to collect an NFT from each template assigned to a Set. For example, let's imagine we are distributing football card NFTs. A Set might be all of the NFTs for a player on a given team in the 2022 season.
```

Comment:
This is sorta pediatric but, I don't think we want a user to collect every single edition/NFT/Collectable from every nft_template in a Set. We want the user to collect a single edition/NFT/Collectable from every nft_template in a Set.